### DISCUSSION TITLE
and/or in level 4

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#1250] [Archive]
Level 4 now supports `and` but not `or`. This was introduced a million years ago (really like in Jan 2020 or something) because it enabled rock paper scissors and I had no better ideas at the time for adventures, see:

https://www.hedycode.com/hedy/4#fortune

Now that we are designing the levels more properly the question remains what to do with this. Should `and` and `or` be a separate level? I think it is fun and enables all sorts of things, and could fit somewhere early-on. Of course, we can only support 1 level like this: `x or y` but not `(`x` or `y`) and `z`` because kids will not have learned brackets (but that is ok because nested boolean operations are hard, and if kids really really need it they cold store it in var which would work.

What do you think @MarleenGilsing and @OnnoEbbens?

**Comment by MarleenGilsing** [comment]
If we want the students to notice it, we definitely should put it in a seperate level. Because level 4 is already a lot of new material with the if command (and else) and we don't want too much cognitive load in the early levels.

**Comment by Felienne** [comment]
Yes, let's put this in a separate level! I have to think a bit about where, I think after the for loops (but before while) maybe because it is kind of complex.

**Comment by Felienne** [comment]
Closed by #1277 and now [#1331](https://github.com/Felienne/hedy/pull/1331)
